src/types.ts

```diff
@@ -248,6 +345,7 @@ type PaymentMethod = {
 export enum FeedbackEvent {
```

Comment:
Not required, but is there a way to pass a "custom" feedback event type?

Reply:
We were checking for specific event values (enum), but it would make sense to use the string type.